src/saga_exec.rs

```diff
@@ -840,24 +836,22 @@ impl<UserType: SagaType> SagaExecutor<UserType> {
             let sgaction = if live_state.injected_errors.contains(&node_id) {
                 Arc::new(ActionInjectError {}) as Arc<dyn Action<UserType>>
             } else {
-                Arc::clone(
-                    live_state
-                        .saga_template
-                        .launchers
-                        .get(&node_id)
-                        .expect("missing action for node"),
-                )
+                // TODO(AJS) - don't unwrap
```

Comment:
We already had an `expect` here in the prior case. This feels like we need to make some adjustments to the wrapping function/API in order to allow this to fail safely.

Reply:
Ah, sorry, that comment really should be on the following line. Now that the registry is separate, looking up the action can fail more easily. However, I'm actually unsure if we want to panic here. In this case, with the upgrade strategy we talked about this would only occur if the code changed, which would definitely be an invariant violation. It could also occur if we never ran the sagas during tests, but that seems unlikely.

Reply:
I just went ahead and removed the TODO. It's not an issue based on our current usage.